src/libstore/globals.hh

```diff
@@ -881,6 +881,19 @@ public:
 
     Setting<std::string> flakeRegistry{this, "https://github.com/NixOS/flake-registry/raw/master/flake-registry.json", "flake-registry",
         "Path or URI of the global flake registry."};
+
+    Setting<bool> ignoreSymlinkStore{
+        this, false, "ignore-symlink-store",
```

Comment:
I'm using `ignore-symlink-store` here to be consistent with the environment variable. However, there are arguably better names for this config option. I'm happy to change it if you have any suggestions.

Reply:
For example, one possible option could be `allow-symlink-store`.

Reply:
:+1: on `allow-symlink-store` (or maybe `allow-symlinked-store`).

Reply:
Done.